maxim/logger/mistral/client.py

```
@@ -0,0 +1,215 @@
+            scribe().warning(
+                f"[MaximSDK][MaximMistralChat] Error in generating content: {e}"
+            )
+
+        response = self._chat.complete(*args, **kwargs)
+
+        try:
+            if generation is not None:
+                generation.result(MistralUtils.parse_completion(response))
+            if is_local_trace and trace is not None:
+                if response.choices:
+                    text = MistralUtils._message_content(response.choices[0].message)
```

Pylint (3.3.7)

[warning] Access to a protected member _message_content of a client class

(W0212)